holidays/utils.py

```diff
@@ -949,7 +1000,7 @@ def jd2date(self, jd: float) -> datetime:
 
         h = int(f * 24)
         m = int((f - h / 24.0) * 1440)
-        s = round((f - h / 24.0 - m / 1440.0) * 86400.0)
+        s = round((f - h / 24.0 - m / 1440.0) * 86400.0, 2)
```

Comment:
Why round to 2 digits then make integer instead of one step? `round(n, 0)` return `int`

Reply:
I was getting values above `59sec` for some of the tests like some 1999 dates ... You can check with the following JD `2451258.574303147` 
Another thing is, we should keep this function as close as possible to the main implementation, it's easier to check other language implementations and compare it with our Python version.
Anyway, if you have any suggestion that fixes this without breaking the rest of the tests, I'll be happy to include it ;)

Reply:
Yes, I checked and I agree :)
But I have more general question about using Meeus. We already have `pymeeus` in dependencies (from `convertdate`). Maybe use needed functions from it and don't add more code will be better? @dr-prodigy, @arkid15r, what do you think about it?

Reply:
I was checking the dependencies for version pinning proposal and I've found that too ... pymeeus will do the trick and it's already a dependency (from convertdate as mentioned by @KJhellico )
So maybe it's better to remove the code related to Meeus Algorithms :'( ... I'll close this PR in favour of a new PR that I will open using pymeeus ... if this is OK for everyone ;)